### PR TITLE
Add recipe for hmmlearn

### PR DESCRIPTION
**Pull request by mcg1969**

### CODE REVIEW

**Comment by mcg1969** [comment]
@superbobry I am building this conda-forge recipe to facilitate conda package building for your HMMLearn package. Would you mind if I listed you as a maintainer for this recipe? This allows conda-forge to notify you if there are issues with this package, but it will _not_ open you up to all sorts of email on unrelated packages.

File: recipes/hmmlearn/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/hmmlearn/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/hmmlearn/meta.yaml]: It looks like this compiles a C-extension so please include `toolchain` as a `build` requirement. Also see the following about building against numpy:

https://conda-forge.org/docs/meta.html#building-against-numpy

File: recipes/hmmlearn/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - python
+    - setuptools
+    - numpy
+  run:
+    - python 
+    - numpy
+    - scipy
+    - scikit-learn
```
Comment (synapticarbors) [inline recipes/hmmlearn/meta.yaml]: It looks like there is a version restriction on sklearn:

https://github.com/hmmlearn/hmmlearn/blob/master/setup.py#L49

File: recipes/hmmlearn/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+test:
+  imports:
+    - hmmlearn
```
Comment (synapticarbors) [inline recipes/hmmlearn/meta.yaml]: maybe also import `hmmlearn.base` as a test that the C-extension was built.

**Pull request closed without merging**